docker/kubernetes/templates/tubemq-master-ini-configmap.yaml

```
@@ -38,7 +38,7 @@ data:
     metaDataPath=/data/meta_data
     useWebProxy=true
 
-    [zookeeper]
+    [meta_zookeeper]
     zkNodeRoot=/tubemq
     zkServerAddr={{ template "inlong.fullname" . }}-{{ .Values.zookeeper.component }}:2181
```

The `inlong.zookeeper.hostname` configuration can be used.

```suggestion
    zkServerAddr={{ template "inlong.zookeeper.hostname" . }}:{{ .Values.zookeeper.ports.client }}
```